docker/resources/inmanta-env

```diff
@@ -0,0 +1,11 @@
+# Loading the content of the server env file if it is there
```

Comment:
When I looked at the name of this file for the first time, I though it included a list of environment variables, but that's not the case. Could you rename the file to `load-inmanta-env` or something similar as such that it's immediately clear that this file takes case of the loading part.

Reply:
Done